src/Compilers/CSharp/Portable/Parser/DirectiveParser.cs

```diff
@@ -32,7 +32,8 @@ public CSharpSyntaxNode ParseDirective(
             bool isActive,
             bool endIsActive,
             bool isAfterFirstTokenInFile,
-            bool isAfterNonWhitespaceOnLine)
+            bool isAfterNonWhitespaceOnLine,
+            bool trailing)
```

Comment:
>trailing [](http://example.com/codeflow?start=17&length=8)

Do we really need this additional information that looks redundant? Isn't ```isAfterNonWhitespaceOnLine``` sufficient?